sql/pgwire_test.go

```diff
@@ -364,6 +365,15 @@ func TestPGPreparedQuery(t *testing.T) {
 				time.Date(2006, 7, 8, 0, 0, 0, 0, time.FixedZone("", 0)),
 			),
 		},
+		"SELECT $1::date, $2::timestamp": {
+			base.Params(
+				time.Date(2006, 7, 8, 0, 0, 0, 0, time.FixedZone("", 0)),
+				time.Date(2001, 1, 2, 3, 4, 5, 0, time.FixedZone("", 0)),
+			).Results(
+				time.Date(2006, 7, 8, 0, 0, 0, 0, time.FixedZone("", 0)),
+				time.Date(2001, 1, 2, 3, 4, 5, 0, time.FixedZone("", 0)),
+			),
+		},
```

Comment:
Because it doesn't test different code paths. The error wasn't in performing an insert, but in passing `time.Time` to `sql.{DB.Tx}.Exec`.